controls/cis_rhel8.yml

```diff
@@ -971,15 +971,14 @@ controls:
       - service_xinetd_disabled
 
   - id: 2.2.20
-    title:  Ensure rsync is not installed or the rsyncd service is masked (Automated)
+    title: Ensure X window server services are not in use (Automated)
     levels:
-      - l1_server
-      - l1_workstation
+      - l2_server
     status: automated
+    notes: >-
+      The rule also configures correct run level to prevent unbootable system.
     rules:
-      - package_rsync_removed
-    related_rules:
-      - service_rsyncd_disabled
+      - xwindows_remove_packages
```

Comment:
This rule removes 3 packages but the policy asks to remove only `xorg-x11-server-common`.
Wouldn't be better to select `package_xorg-x11-server-common_removed` here?